libs/langgraph/langgraph/pregel/remote.py

```diff
@@ -0,0 +1,360 @@
+from typing import (
```

Comment:
should this live in the SDK?

Reply:
I didn't consider it, but that's a fair thought.

1. If I think of `RemotePregel` as a client of the SDK, then it makes sense to live in the SDK.
2. If I think of `RemotePregel` as a primitive of `langgraph` or as a "prebuilt" (kind of like a `ToolNode`), then it feels more appropriate to live in `langgraph` and using the SDK is just an implementation detail. For example, we could just use vanilla `requests` to make the remote API calls.

I don't have a strong opinion. I could go either way.

Reply:
I think it needs to be in langgraph